test/SimpleCraApp.E2ETests/package.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "simplecraapp.e2etests",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "cy:open": "cypress open",
+    "cy:run": "cypress run"
+  },
+  "author": "",
+  "license": "ISC",
```

Technically the upper package has an MIT license, so we should update this one to match

```suggestion
  "license": "MIT",
```